server/src/main/java/org/jetbrains/bsp/bazel/server/sync/IntelliJProjectTreeViewFix.kt

```diff
@@ -9,29 +10,77 @@ import org.jetbrains.bsp.bazel.workspacecontext.WorkspaceContext
 import java.net.URI
 import java.nio.file.Files
 import java.nio.file.Paths
+import kotlin.io.path.Path
+import kotlin.io.path.name
+import kotlin.io.path.toPath
+import kotlin.io.path.listDirectoryEntries
+import kotlin.io.path.isSymbolicLink
+import kotlin.io.path.readSymbolicLink
 
-class IntelliJProjectTreeViewFix(private val bazelPathsResolver: BazelPathsResolver) {
+class IntelliJProjectTreeViewFix(
+    private val bazelPathsResolver: BazelPathsResolver,
+    private val bazelInfo: BazelInfo
+) {
 
     fun createModules(
-        workspaceRoot: URI, modules: Sequence<Module>, workspaceContext: WorkspaceContext
-    ): Sequence<Module> = if (isFullWorkspaceImport(workspaceContext)) {
-        createWorkspaceRootModule(workspaceRoot, modules)
-    } else {
-        createMultipleModules(workspaceContext, workspaceRoot, modules)
+        workspaceRoot: URI,
+        modules: Sequence<Module>,
+        workspaceContext: WorkspaceContext
+    ): Sequence<Module> {
+        return if (isFullWorkspaceImport(workspaceContext, workspaceRoot)) {
+            createWorkspaceRootModule(workspaceRoot, modules)
+        } else {
+            createMultipleModules(workspaceContext, workspaceRoot, modules)
+        }
     }
 
-    private fun isFullWorkspaceImport(workspaceContext: WorkspaceContext): Boolean =
-        importTargetSpecs(workspaceContext).any { it.startsWith("//...") }
+    private fun isFullWorkspaceImport(workspaceContext: WorkspaceContext, workspaceRoot: URI): Boolean {
+        return importTargetSpecs(workspaceContext).any { it.startsWith("//...") } ||
```

Comment:
```suggestion
        return importTargetSpecs(workspaceContext).any { it.startsWith("//...") || it.startsWith("@//...") } ||
```

we need it for bazel 6 i think